src/imgbased/command.py

```diff
@@ -31,10 +30,14 @@ def nsenter(arg, new_root=None, shell=False, environ=None):
     environ = environ or os.environ
     log.debug("Executing: %s", arg)
     proc = subprocess.Popen(arg, stdout=subprocess.PIPE, env=environ,
-                            stderr=DEVNULL, shell=shell).communicate()
-    ret = proc[0]
-    log.debug("Result: %s", repr(ret))
-    return ret
+                            stderr=subprocess.PIPE, shell=shell)
+    comm = proc.communicate()
+    stdout = comm[0]
+    stderr = comm[1]
+    log.debug("STDOUT: %s", repr(stdout))
+    log.debug("STDERR: %s", repr(stderr))
+    log.debug("ReturnCode: %s", repr(proc.returncode))
+    return stdout
```

Comment:
Just wondering, is the return consumed?

If the stdout is printed here and then again printed by the caller, we will see the same log twice.

Reply:
It's not, but I wanted to keep the functionality intact.

Reply:
Ok, at least is not printed twice :+1:

Reply:
Keeping the behavior the same is good choice for this change.